Cogs/AdminCommands.py

```diff
@@ -155,3 +156,63 @@ async def stop(self, ctx):
         if await confirmation(self.bot, ctx):
             await ctx.send('Stopping...')
             exit(0)
+
+    @commands.command()
+    @commands.has_permissions(administrator=True)
+    async def stats(self, ctx):
+        """Outputs various stats of the server
+        Send message with server stats to user
+
+        Outputs:
+            Message to chat including total number of channels, text channels, voice channels, users, classes, roles,
+            and people with top ten roles.
+        """
+
+        guild=ctx.guild
+
+        embed=discord.Embed(
+        title="Server Stats",
```

Comment:
Again, minor formatting consistency: indent this line so all parameters are the same indentation level